_posts/patterns/2016-03-01-dropdown-menu.md

```diff
@@ -0,0 +1,32 @@
+---
+layout:                 pattern
+title:                  Dropdown Menu
+date:                   2016-02-29 00:00:00
+
+categories:             patterns
+tags:
+- atomic
+- pattern
+- js
+- dropdown
+- menu
+
+slug:                   dropdown menu
+
+url_styles:             "patterns/_dropdown-menu.scss"
+
+description:            Dropdown Menu in a view.
+
+info:                   A JavaScript controlled dropdown menu.
+ui-toolkit-component:   dropdown-menu
+js:                     "/examples/dropdown-menu-js.html"
+---
+<h4 class="hd-6 example-set-hd">Basic Dropdown</h4>
+<div class="example-set">
+	<div class="js-user-cta">Hook for dropdown menu</div>
```

Comment:
What does ``Hook for dropdown menu`` mean? Maybe it should be a comment? It seems a bit strange to have it rendered before the JavaScript evaluates.